applications/endoscopy_tool_tracking/python/README.md

```diff
@@ -50,3 +51,9 @@ the working directory.
     cd <HOLOHUB_BUILD_DIR>
     python3  <HOLOHUB_SOURCE_DIR>/applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py --source=aja
     ```
+
```

Comment:
@jjomier is this branch using the latest code?  The instructions for running with deltacast capture card seems to be missing?

Reply:
Deltacast is not supported for Python (only C++)

Reply:
I see, thanks!